binaryRepresentation/binaryRepresentation.c

```diff
@@ -0,0 +1,83 @@
+﻿#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <locale.h>
+#include "binaryRepresentation.h"
+#define NUM_SIZE (sizeof(int) * 8)
+
+void printArray(int arrayOfIntegers[], int lenghtOfArray) {
+    for (int i = 0; i < lenghtOfArray; i++)
+        printf("%d", arrayOfIntegers[i]);
+    printf("\n");
+}
+
+void numberToBinaryFromDecimal(int* array, int number) {
+    int mask = 1;
+    for (int i = NUM_SIZE - 1; i >= 0; i--) {
+        if ((number & mask) == 0) {
+            array[i] = 0;
+        }
+        else {
+            array[i] = 1;
+        }
```

Comment:
```suggestion
        array[i] = ((number & mask) == 0) ? 0 : 1;
```
Тоже писал в замечании, тоже проигнорировали

Reply:
А если бы array был булевым, то просто `(number & mask) == 0`